Sources/TuistKit/ProjectEditor/ProjectEditor.swift

```diff
@@ -120,4 +144,61 @@ final class ProjectEditor: ProjectEditing {
         try writer.write(workspace: descriptor)
         return descriptor.xcworkspacePath
     }
+
+    /// Attempts to load the plugins at the given path. If unable to find the Config manifest
+    /// displays a warning to the user.
+    /// - Returns: The loaded `Plugins`.
+    private func loadPlugins(at path: AbsolutePath) -> Plugins {
+        do {
+            let config = try configLoader.loadConfig(path: path)
+            return try pluginService.loadPlugins(using: config)
```

Comment:
I don't think it should be the responsibility of the editor to load the config and the plugins. That should be done by the edit service and passed down to this class.

Reply:
Good point! Ill move this up the stack to the `EditorService`